src/care/array_utils.h

```diff
@@ -519,7 +549,8 @@ inline int BinarySearch(const T *map, const int start,
       return -1 ;
    }
 #ifdef CARE_DEBUG
-   checkSorted(&(map[start]), mapSize, "BinarySearch", "map") ;
+   const bool allowDuplicates = true;
```

Comment:
the comment change on L 478 and this line of code appear to conflict with each other.

Reply:
I think this is right. This is in BinarySearch. If I understand correctly, BinarySearch does allow duplicates (it will find an element with the desired value, and if others with the same value exist nearby they will be adjacent, either before or after)

However, IntersectArrays does not allow duplicates, because we have to return the indicies of all the matches and that would be with duplicates.